solr/core/src/java/org/apache/solr/servlet/LoadAdminUiServlet.java

```diff
@@ -24,6 +24,7 @@
 import org.apache.solr.core.CoreContainer;
 import org.apache.solr.core.SolrCore;
 
+import javax.servlet.ServletException;
```

Comment:
This looks like an unused import to me?

Reply:
@madrob Should have been against master originally. I thought it was but it's been a long time since I was looking at this one. I'll check and start over.

Reply:
oof. You're right. I'll open a new PR.